src/org/labkey/test/tests/component/GridPanelViewTest.java

```diff
@@ -644,8 +644,10 @@ private void testEditView(String testName, String viewName)
 
         validateGridHeader(testName, grid, EDITED_ALERT, true);
 
-        checker().verifyTrue("Filter was removed but number of rows in grid did not increase.",
-                grid.getRows().size() > rowCount);
+        // Wait until grid.getRows().size() gets an updated count.
+        checker().wrapAssertion(()->
+                waitFor(()->grid.getRows().size() > rowCountBefore,
+                        "Filter was removed but number of rows in grid did not increase.", 5_000));
```

Comment:
`waitFor` doesn't throw an exception that `checker()` knows how to deal with by default.
```suggestion
        checker().addRecordableErrorType(WebDriverException.class).wrapAssertion(()->
                waitFor(()->grid.getRows().size() > rowCountBefore,
                        "Filter was removed but number of rows in grid did not increase.", 5_000));
```

Reply:
Had to modify slightly, addRecordableErrorType has a void return.